cspell.json

```diff
@@ -225,6 +225,9 @@
         "**/.gitignore",
         ".vscode/**",
         "**/.idea/**",
-        "hermes/wasm/*/**"
+        "hermes/wasm/*/**",
+        "**/*.pest",
+        "**/*.cddl",
+        "**/tests/**",
```

Comment:
Don't add these to the ignore path.
They should be spell-checked.
Spelling errors should be corrected, and you should be able to add comments to both the `.pest` and `.cddl` files so that it ignores words specific for those files.

Reply:
I removed these ignores in the config, and added specific words to each file instead.